Modules/Quotas/src/Quotas.Application/Metrics/UsedFileStorageSpaceMetricCalculator.cs

```diff
@@ -13,7 +13,7 @@ public UsedFileStorageSpaceMetricCalculator(IFilesRepository filesRepository)
 
     public async Task<uint> CalculateUsage(DateTime from, DateTime to, string uploader, CancellationToken cancellationToken)
     {
-        var usedSpace = await _filesRepository.UsedSpace(uploader, from, to, cancellationToken);
-        return usedSpace;
+        var usedSpace = await _filesRepository.AggregateUsedSpace(uploader, from, to, cancellationToken);
+        return (uint)(usedSpace / 1024 / 1024);
```

Comment:
I don't know what the max positive value of uint and long are, but if the max value of long is greater than the one of uint, you should do a check first and throw an Exception if it doesn't fit. Otherwise we may have a loss here